scripts/bootstrap.js

```diff
@@ -97,8 +97,8 @@ function run() {
       command: () => {
         log.info(prefix, 'prepare');
         spawn(
-          `lerna run prepare ${
-            process.env.CI ? `--concurrency ${maxConcurrentTasks} --stream` : ''
+          `nx run-many --target=prepare --all --parallel ${
+            process.env.CI ? `--max-parallel=${maxConcurrentTasks}` : ''
```

Comment:
Not sure what NX provides that you need (the PR description isn't telling me much) but once https://github.com/storybookjs/storybook/pull/13907 is merged this can be
```
yarn workspaces foreach -pt run prepare
```
without having to setup and maintain more config files